README.md

```diff
@@ -131,9 +132,9 @@ query {
 }
 
 query {
-  participant(id: 1) {
+  participant(uuid: <valid_participant_uuid_string_from_previous_query_result>) {
```

Comment:
ok, I see what you mean now by the query not being runnable as is. Maybe add a random UUID in these examples, though consistent if a participant is a delegate on another query etc, and add a note about needing to replace them with real ones before the examples (next to `Then run some queries:`)

Reply:
hmm... but the consistency with the previous query will not be obvious, since we don't include the query results in the examples (though we could).

Reply:
I see what you mean - maybe just make it a little shorter?  Like `<participant uuid fetched in previous query result>`

Reply:
that's longer :)

Reply:
?
```
<valid_participant_uuid_string_from_previous_query_result>
<participant uuid fetched in previous query result>
```